<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,11,1">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>INSERT/MODIFY PERSON</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6,5,1">
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>200</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="ipt_person"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_search">
           <property name="text">
            <string>SEARCH</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>LAST NAME</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>FIRST NAME</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPEOPLE">
    <property name="title">
     <string>PEOPLE</string>
    </property>
    <addaction name="actionABOUT"/>
   </widget>
   <widget class="QMenu" name="menuTRAILS">
    <property name="title">
     <string>TRAILS</string>
    </property>
    <addaction name="actionTODO"/>
   </widget>
   <widget class="QMenu" name="menuRESULTS">
    <property name="title">
     <string>RESULTS</string>
    </property>
    <addaction name="actionTODO2"/>
   </widget>
   <addaction name="menuPEOPLE"/>
   <addaction name="menuTRAILS"/>
   <addaction name="menuRESULTS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionABOUT">
   <property name="text">
    <string>ABOUT</string>
   </property>
  </action>
  <action name="actionTODO">
   <property name="text">
    <string>TODO</string>
   </property>
  </action>
  <action name="actionTODO2">
   <property name="text">
    <string>TODO2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_search</sender>
   <signal>clicked()</signal>
   <receiver>ipt_person</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>print()</slot>
 </slots>
</ui>
